From 5083ba09e87aab650d84764a06d679fbb338e718 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Tue, 19 Dec 2023 16:19:19 +0530
Subject: [PATCH] drivers: clk: qcom: Initialize default return value

Initialize default return value NOTIFY_DONE for the notifier
call function cpu_clk_notifier_fn to fix static analysis error.

Change-Id: Iaffcc3aa5d519e097603ef24fa7f19909d6a48c6
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 drivers/clk/qcom/apss-ipq6018.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/qcom/apss-ipq6018.c b/drivers/clk/qcom/apss-ipq6018.c
index d40cd0454181..ea0305b83b33 100644
--- a/drivers/clk/qcom/apss-ipq6018.c
+++ b/drivers/clk/qcom/apss-ipq6018.c
@@ -90,7 +90,7 @@ static const struct qcom_cc_desc apss_ipq6018_desc = {
 static int cpu_clk_notifier_fn(struct notifier_block *nb, unsigned long action,
 				void *data)
 {
-	int err;
+	int err = NOTIFY_DONE;
 
 	if (action == PRE_RATE_CHANGE)
 		err = clk_rcg2_mux_closest_ops.set_parent(&apcs_alias0_clk_src.clkr.hw,
-- 
2.34.1

